### PR TITLE
Aatrey882 !lmgfty

### PR DESCRIPTION
**Pull request by aatrey882**
Slack will return either lmgtf(Let me google for you) or google search result link...
Correct regex has been modified.

### CODE REVIEW

File: Parsers/!lmgtfy or !google search term.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
+activation_example:!lmgtfy or !google search term 
+regex:!lmgtfy
```
Comment (earlduque) [inline Parsers/!lmgtfy or !google search term.js:3]: description says lmgtfy or google, but only parses for lmgtfy

File: Parsers/!lmgtfy or !google search term.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/*
```
Comment (earlduque) [inline Parsers/!lmgtfy or !google search term.js:1]: please remove the exclamation points from the filename

File: Parsers/!lmgtfy or !google search term.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+} else if (current.text.indexOf('!google') > -1) {
+    where = current.text.indexOf('!google') + 8;
+    which = 'google';
+}
+
+var term = current.text.substr(where).trim();
+
+if (term == '') {
+    var send_confusion = new x_snc_slackerbot.Slacker().send_chat(current.channel, ':upside_down_face: !docs *something*', false, '', current.thread_ts);
```
Comment (earlduque) [inline Parsers/!lmgtfy or !google search term.js:24]: please update this exception handler to not reference !docs and properly reference the intent of this parser

**Review by earlduque — CHANGES_REQUESTED** [review]

**Review by aatrey882 — COMMENTED** [review]
Hi @earlduque , I've done the changes

File: Parsers/!lmgtfy or !google search term.js (hunk 2 above)
Comment (aatrey882) [reply Parsers/!lmgtfy or !google search term.js]: Thanks @earlduque, I did not know about it, I've removed it

**Review by earlduque — APPROVED** [review]

**Pull request merged**